test/openAPI/features/support/helpers/helpers.js

```diff
@@ -1,3 +1,33 @@
 module.exports = {
   localhost: 'http://localhost:3333/',
+  contentTypeHeader: {
+    key: 'content-type',
+    value: 'application/json; charset=utf-8',
+  },
+  defaultExpectedResponseTime: 15000,
+  oauthTokenEndpoint: 'oauth/token',
+  //shares
+  clientResponseSchema: {
+    type: 'object',
+    properties: {
+      responseTime: { type: 'string' },
+      response: {
+        type: 'object',
+        properties: { clientId: { type: 'string' } },
+        errors: {
+          type: 'array',
+          items: {
+            type: 'object',
+            properties: {
+              errorCode: {
+                type: 'string',
+              },
+              errorMessage: { type: 'string' },
+            },
+            additionalProperties: false,
+          },
+        },
+      },
+    },
+  },
```

Comment:
This is not the response schema for /outh/token. 
<img width="439" alt="Zrzut ekranu 2023-05-11 o 09 47 02" src="https://github.com/GovStackWorkingGroup/bb-identity/assets/80507519/022c0d1d-990b-442f-9b7c-9be4313be82a">

And another for status 400:
<img width="394" alt="Zrzut ekranu 2023-05-11 o 09 47 35" src="https://github.com/GovStackWorkingGroup/bb-identity/assets/80507519/0039fa44-9af7-4c61-aa3b-174f1470d1c8">